files/build_templates/docker_image_ctl.j2

```diff
@@ -366,7 +366,12 @@ wait() {
 }
 
 stop() {
+{%- if docker_container_name == "teamd" %}
+    # Longer timeout of 60 sec to wait for Portchannels to be cleaned.
+    docker stop -t 60 {{docker_container_name}}$DEV
```

Comment:
@judyjoseph what about default systemd service timeout (90 sec)? Do we need to align also?

Reply:
@nazariig will prefer to keep it in sync with master, but we can tune this. 
During my tests I found the cleanup was quick in 201911 and hence even 60sec is a high  value. But if you find that the issue #6537, where you have a large number of POs it takes more time - we can increase it - is that ok ?

Reply:
@judyjoseph ok. Thanks for the clarifications.